tests/resources/aria-at-test-run.mjs

```diff
@@ -377,7 +377,12 @@ export function instructionDocument(resultState, hooks) {
                   ]),
                   enabled: behavior.checked,
                   value: behavior.more.value,
-                  focus:
+                  focusImpact:
```

Comment:
The typedef for [`InstructionDocumentAssertionChoiceOptionsOptionsMore`](https://github.com/w3c/aria-at/blob/aa944ce836bffae3f45bbfa7005a82b352c3518c/tests/resources/aria-at-test-run.mjs#L1141) should be updated to include `focusImpact` and `focusDetails`

Reply:
Good catch! Done!

Reply:
Thanks!